app/services/ReviewService.scala

```diff
@@ -32,11 +32,17 @@ class ReviewService @Inject() (publishedService: PublishedService, repository: A
   val logger: Logger = Logger(this.getClass)
 
   def approval2iReviewInfo(id: String): Future[RequestOutcome[ProcessReview]] =
+    getReviewInfo(id, ReviewType2i)
```

Comment:
are we sure that the data for 2i and fact check will not diverge??

Reply:
nothing to say it will diverge and if it does later, we then work out how to handle it.  If data changes, then the screens will change as well so will need to change whole flow.  I think keeping data in same format for now makes sense based on our timescales